src/App/Program.fs

```diff
@@ -77,7 +77,8 @@ type MissionOptions
         tier1OrgsToAdd: int,
         nonTier1NodesToAdd: int,
         randomSeed: int,
-        pubnetParallelCatchupStartingLedger: int
+        pubnetParallelCatchupStartingLedger: int,
+        inMemoryPubnetParallelCatchup: bool
```

Comment:
should the name of the flag not include "pubnet"? It's used in many places, not just pubnet catchup